docs/api/specs/browserResidentDesktopAgents.md

```diff
@@ -0,0 +1,253 @@
+---
+id: browserDesktopAgents
+sidebar_label: Browser Desktop Agents
+title: Browser-Resident Desktop Agents (next)
+---
+
```

Comment:
"must"
Almost all occurrences of the word "must" on this page are capitalized as "MUST". However, if you search you can see there are (I think) 3 occurrences that are just in lower case

Reply:
I haven't changed every instance (some are not requirements, but more like preconditions or precautions (care _must_ be taken)

Reply:
fixed in 976af80d5ba8d831e202aabcd37d1a7886e4c930